.travis.yml

```diff
@@ -9,22 +9,8 @@ python:
 addons:
   postgresql: "9.6"
   apt:
-# only add the two lines below if you need wkhtmltopdf for your tests
-    sources:
-      - pov-wkhtmltopdf
-#    Search your sources alias here:
-#      https://github.com/travis-ci/apt-source-whitelist/blob/master/ubuntu.json
     packages:
-      - expect-dev  # provides unbuffer utility
-      - python-lxml  # because pip installation is slow
```

Comment:
We need lxml in this repo.
You suggest to install it with pip ?

Reply:
lxml is an Odoo [dependency](https://github.com/odoo/odoo/blob/9e8f70e4849b0eeaca8b5cf51372ecfa23dc561b/requirements.txt#L17), so MQT will install automatically, and pip will cache it so performance should not be an issue..